common/api/imodeljs-frontend.api.md

```diff
@@ -4972,6 +4980,18 @@ export interface IpcAppOptions {
     iModelApp?: IModelAppOptions;
 }
 
+// @alpha
+export interface IRealityDataConnection {

```

Comment:
Nitpick: coding guidelines forbid sticking an "I" in the front of a type name.

Reply:
Push a new fix for this:
Rename IRealityDataConnection -> RealityDataConnection
and RealityDataConnection -> RealityDataConnectionImpl